lib/ansible/modules/crypto/luks_device.py

```diff
@@ -188,10 +223,41 @@ class Handler(object):
     def __init__(self, module):
         self._module = module
         self._lsblk_bin = self._module.get_bin_path('lsblk', True)
+        self._blkid_bin = self._module.get_bin_path('blkid', True)
 
     def _run_command(self, command):
         return self._module.run_command(command)
 
+    def get_device_by_uuid(self, uuid):
+        ''' Returns the device that holds UUID passed by user
+        '''
+        uuid = self._module.params['uuid']
+        if uuid is None:
+            return None
+
+        query = "%s=%s" % ("UUID", uuid)
```

Comment:
```suggestion
        query = '%s=%s' % ('UUID', uuid)
```
Since single quotes are used everywhere else.

Reply:
Done, thanks!